site/it/community/contribute/code.md

```diff
@@ -0,0 +1,136 @@
+# Contributo al codice TensorFlow
+
+Sia che tu voglia aggiungere una funzione perdita, migliorare la copertura di un test, oppure scrivere una
+RFC per un cambiamento rilevante di progettazione, questa parte della guida del contributore ti sarà d'aiuto
+per cominciare. Grazie per il lavoro e l'interesse nel miglioramento di TensorFlow.
+
+## Prima di cominciare
+
+Prima di contribuire al codice sorgente di un progetto TensorFlow, cortesemente consulta il file `CONTRIBUTING.md` nel repository GitHub del progetto. (Per esempio, vedi
+[file CONTRIBUTING.md nel repository centrale di TensorFlow](https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md).) A tutti i contributori è richiesto di firmare un [Accordo di Licenza di Contribuzione](https://cla.developers.google.com/clas) (CLA).
+
+Per evitare attività duplicate, prima di cominciare a lavorare su una funzionalità non banale, cortesemente consulta [RFCs correnti](https://github.com/tensorflow/community/tree/master/rfcs) e contatta gli sviluppatori nei forum TensorFlow. Talvolta siamo selettivi quando dobbiamo decidere se aggiungere una funzionalità, ed il modo migliore di contribuire ed aiutare il progetto è lavorare su problemi conosciuti. 
+
+## Temi per il nuovo contributore
+
+I nuovi contributori alla ricerca di un primo tema per un contributo al codice TensorFlow, dovrebbero cercare i seguenti tag. Raccomandiamo fortemente che i nuovi contributori affrontino inizialmente progetti “easy” e problematiche "good first issue"; questo aiuta il contributore a familiarizzarsi con il flusso di lavoro della contribuzione, ed agli sviluppatori del core di conoscere il contributore.
+
+- `good first issue`
+- `easy`
+- `contributions welcome`
+
+Se siete interessati a reclutare un team per aiutarvi ad affrontare un problema ampio o una nuova funzionalità, cortemente mandate una email a [gruppo sviluppatori@](https://groups.google.com/a/tensorflow.org/forum/#!forum/developers) e osservate la nostra lista attuale di RFC. 
```

Comment:
"cortesemente" but in Italian we could skip the translation of please in many cases. See the old good http://tp.linux.it/buona_traduzione.html